src/components/LoginPanel.tsx

```diff
@@ -54,10 +54,12 @@ const LoginPanel: React.FC<{}> = () => {
     const theme = useTheme();
 
     const [loaded, setLoaded] = React.useState(false);
```

Comment:
let's keep this consistent

```suggestion
    const [loaded, setLoaded] = useState(false);
```